### PR TITLE
PT-2088: Improve UI on Pedigree import dialog.

### PR DESCRIPTION
**Pull request by veronikaslc**
Fixed

### CODE REVIEW

**Comment by veronikaslc** [comment]
@allasm @itaiGershtansky  please review

**Comment by itaiGershtansky** [comment]
+1

File: components/pedigree/resources/src/main/resources/pedigree/view/importSelector.js [hunk 1]
```diff
@@ -29,12 +29,13 @@ define([], function(){
...
                     }
                 })        
-                var uploadButton = new Element('input', {type: 'button', 'value': 'Select a local file to be imported', 'class' : 'button'}).wrap('span', {'class' : 'buttonwrapper'});
+                var uploadButton = new Element('input', {type: 'button', 'value': 'Select file', 'class' : 'button'}).wrap('span', {'class' : 'buttonwrapper import-file-button'});
                 uploadButton.observe('click', function(event) {
                     var fileElem = document.getElementById("pedigreeInputFile");
                     fileElem.click();
                 })
-                mainDiv.insert(uploadFileSelector).insert(uploadButton);
+		        var promptImportFile = new Element('div', {'class': 'import-section', }).update("Import file:");
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/view/importSelector.js]: There's a tab here, replace it with spaces.

File: components/pedigree/resources/src/main/resources/pedigree/editor.css [hunk 2]
```diff
@@ -434,6 +434,10 @@
   margin: .2em .2em 0 7.5em;
   display: block;
 }
+.import-file-button {
+  padding-left: 27px;
+  display: table;
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/editor.css]: Don't abuse display types like this, that button is not a table.

**Comment by allasm** [comment]
Cherry-picked into PT-1548 branch

**Pull request closed without merging**